Sources/Subs.php

```diff
@@ -829,6 +829,9 @@ function timeformat($log_time, $show_today = true, $offset_type = false, $proces
 	if ($context['server']['is_windows'] && strpos($str, '%e') !== false)
 		$str = str_replace('%e', ltrim(strftime('%d', $time), '0'), $str);
 
+	// fix unkown %l with %I
```

Comment:
Hm. According to PHP's strftime documentation, supported formats vary from system to system. It all depends on which ones were included in your system's C library:

> Not all conversion specifiers may be supported by your C library, in which case they will not be supported by PHP's strftime().

Additionally, different operating systems deal with unsupported formats differently:

> As the output is dependent upon the underlying C library, some conversion specifiers are not supported. On Windows, supplying unknown conversion specifiers will result in 5 E_WARNING messages and return FALSE. On other operating systems you may not get any E_WARNING messages and the output may contain the conversion specifiers unconverted.

In light of this, we need to come up with something more robust to deal with unsupported formats, because it can vary quite a lot from system to system and we can't assume anything. I've started working on something to do that now, and I will submit it in a PR soon. In the meantime, if you could take this bit out of this PR before we merge it, that'd make it easier for me to avoid merge conflicts when putting together my PR.